jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -146,12 +146,12 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
     [build non-compressedrefs vm (large heap)])])
 
   OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU($build_cpu)
-  if test "x$with_noncompressedrefs" = x; then
-    OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs"
-    OPENJ9_LIBS_SUBDIR=compressedrefs
-  else
+  if test "${with_noncompressedrefs+set}" = set || test "x$OPENJDK_TARGET_CPU_BITS" = x32; then
```

Comment:
This no longer includes the case where `with_noncompressedrefs is set but empty. I think the test should be
```
test "x$with_noncompressedrefs" != x -o "x$OPENJDK_TARGET_CPU_BITS" = x32
```
Note also the use of `-o` (or) rather than two invocations of `test`.

Reply:
Already Fixed.

Reply:
There is something wrong with test "x$with_noncompressedrefs" != x -o "x$OPENJDK_TARGET_CPU_BITS" = x32 in compilation.